build/Linux+BSD/portable/copy-libs

```diff
@@ -79,8 +79,9 @@ getCrossPlatformDependencies() {
   copyLib libQt5Sql.so.5
   copyLib libQt5Svg.so.5
   #copyLib libQt5WebChannel.so.5  # Not present on Debian, and apparently not needed
-  copyLib libQt5WebKit.so.5
-  copyLib libQt5WebKitWidgets.so.5
+  copyLib libQt5WebEngine.so.5
+  copyLib libQt5WebEngineCore.so.5
+  copyLib libQt5WebEngineWidgets.so.5
```

Comment:
The check-depends https://travis-ci.org/ericfont/MuseScore/jobs/123330905#L4731-L4744 reports:<pre># Provided by neither: 13
libcups.so.2
libEGL.so.1
libicudata.so.56
libicui18n.so.56
libicuuc.so.56
libQt5Bluetooth.so.5
libQt5Nfc.so.5
libQt5QuickParticles.so.5
libQt5WebChannel.so.5
libQt5WebEngine.so.5
libQt5WebEngineCore.so.5
libQt5WebEngineWidgets.so.5
libQt5WebSockets.so.5</pre>
Now I've just added WebEngine, Core, & Widgets, and I've updated the version numbers for icu, so those errors will go away on next build.  And we've already decided libQt5QuickParticles is not needed.  But what about these other libraries?  I'm guessing libQt5Bluetooth.so.5 and libQt5Nfc.so.5 won't be needed.  But what about libQt5WebSockets?